doc/en/user/source/community/ogc-api/features/index.rst

```diff
@@ -0,0 +1,202 @@
+OGC API Features
+================
+
+An `OGC Features API <https://github.com/opengeospatial/ogcapi-features>`_ up to date with the  released version of the Features API, along with implementation of the `draft CQL filtering extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/cql>`_ and the `draft multi-crs extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/crs>`__.
+
+Service
+-------
+
+The OGC API Features Service is accessed via the :guilabel:`FEATURES` version :guilabel:`1.0` link on the home page.
+
+Capabilities
```

Comment:
The term "capabilities" is gone from the OGC APIs. If I understand correctly what you're trying to describe here, a better title may be "resources"?

Reply:
Ah, screenshots are nice of course but Michael work is going to invalidate them quite soon :-(

Reply:
Yes I will need to update the screen shots :)

Reply:
It says `Capabilities` in the OpenAPI docs so I was matching that.
![image](https://user-images.githubusercontent.com/629681/108380135-6c04a980-71bb-11eb-8883-401ec26f460d.png)